rpcclient/infrastructure.go

```diff
@@ -1276,6 +1302,10 @@ func New(config *ConnConfig, ntfnHandlers *NotificationHandlers) (*Client, error
 				return nil, err
 			}
 			start = true
+			wsConn.SetPongHandler(func(data string) error {
+				log.Infof("pong received with data: '%s'", data)
```

Comment:
Leftover debug?  I don't think this should be output to the log as Info. If anything, it would be `Debugf`, but I don't really think it's needed at all.